docs/business-finder-link-based-facets.md

```diff
@@ -0,0 +1,51 @@
+# EU Exit Business Readiness finder link-based facets
+
+This document explains why the [EU Exit Business Readiness Finder](https://www.gov.uk/find-eu-exit-guidance-business) ("business finder") uses [content-tagger](https://github.com/alphagov/content-tagger) 'links' to filter the finder.
+
+## How the business finder is different to other finders
+
+### From the user perspective
+
+In most finders, search results get more granular as you select values in different facets (the number of results goes down). The business finder search results get more _broad_ as you select values in different facets (number of results goes up).
+
+The business finder can be sorted by topic ("grouped display"). In this context, a 'topic' is the label of the facet (such as 'Personal data'), except for the 'Sector / Business area' facet, where the _facet value_ (such as 'Aerospace') is the topic.
+
+![Screenshot of 'Sort by Topic'](https://user-images.githubusercontent.com/93511/52811122-38eb2500-308c-11e9-843f-4df29b719235.png)
+
+This view only applicable to the business readiness finder, but is not yet a candidate for deletion, as it's been proven to be a useful feature.
+
+### From the content designer perspective
+
+In most finders, facets are published as part of the content item itself. This means when the content item is updated via its publishing application, the facets are updated too and associated with the change note accompanying the change. This triggers an email to subscription lists which are subscribed to that facet, and the change note appears in the email body.
```

Comment:
Perhaps replace "subscription lists" with subscribers.

Reply:
Have changed to `This triggers an email to subscribers who are subscribed to a subscription list that includes that facet, and the change note appears in the email body.` 👍

Reply:
That's bit wordy. I'd try reading the sentence out loud. 

When writing this documentation, think about what a someone developing in finder-frontend needs to know. They don't need to about change notes as finder-frontend doesn't touch them.

I don't think describing this section as the "content designer" perspective is right, so I question the need for this section. When it comes to how the facets are rendered, the user perspective and the content designer perspective are the same.